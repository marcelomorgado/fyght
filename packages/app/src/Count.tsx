import * as React from "react";

const Count: React.FunctionComponent<{
  count: number;
}> = props => {
  return <h1>{props.count}</h1>;
};

export default Count;
