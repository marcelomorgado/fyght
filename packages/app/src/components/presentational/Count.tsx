import React from "react";

type Props = {
  count: number;
};

export const Count = ({ count }: Props) => <h1>{count}</h1>;
