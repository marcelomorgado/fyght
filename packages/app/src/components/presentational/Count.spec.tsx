import * as React from "react";
import { shallow } from "enzyme";

import { Count } from "./Count";

describe("Count", () => {
  test("renders the heading", () => {
    expect(shallow(<Count count={1} />)).toMatchSnapshot();
  });
});
