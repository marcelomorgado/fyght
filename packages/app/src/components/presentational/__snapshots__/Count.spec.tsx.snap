// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Count should render the component 1`] = `
<h1>
  1
</h1>
`;
